### PR TITLE
Warmboot

### PR DESCRIPTION
**Pull request by jaymzh**

### CODE REVIEW

File: fboss/agent/platforms/wedge/WedgePlatform.cpp [hunk 1]
```diff
@@ -137,7 +137,7 @@ void WedgePlatform::initLocalMac() {
   // For now, we take the MAC address from eth0, and enable the
   // "locally administered" bit.  This MAC should be unique, and it's fine for
   // us to use a locally administered address for now.
-  std::vector<std::string> cmd{"/sbin/ip", "address", "ls", "eth0"};
+  std::vector<std::string> cmd{"/sbin/ip", "address", "ls", "ma1"};
```
Comment (sonoble) [inline fboss/agent/platforms/wedge/WedgePlatform.cpp:140]: ma1 is ONL specific .. do you we want to push this in the generic repo?

Comment (oridb) [inline fboss/agent/platforms/wedge/WedgePlatform.cpp:140]: I'm going to push a change to make it configurable.

Comment (jaymzh) [inline fboss/agent/platforms/wedge/WedgePlatform.cpp:140]: @sonoble  yeah the plan here is to have you add `--interface ma1` (or similar) to your init scripts - this diff is just to get you able to build... we'll merge the full fixes internally and push

**Comment by oridb** [comment]
This has been solved with a different patch -- now there's a `--mgmt_if` option.

**Comment by sonoble** [comment]
Fantastic, thanks!

**Pull request closed without merging**